dataeng/resources/prefect-flows-deployment.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+set -ex
+
+# Creating python3.8 virtual env
+PLATFORM_VENV="platform_venv"
```

Comment:
Is this copy/paste? We aren't using the edx-platform in this job

Reply:
yeah it was typo